src/Hazelcast.Net/Messaging/ClientMessageConnection.cs

```diff
@@ -128,9 +128,6 @@ private bool ReceiveMessageBytesAsync(SocketConnectionBase connection, IBufferRe
                 }
             }
 
-            // update the reference
-            bufferReference.Buffer = bytes;
```

Comment:
What's reason to keep update the reference? I don't see any change.

Reply:
guess what, I wondered the same, then realized that we *do* change it in fact, I am going to add a comment in the code to explain